peer/peer.go

```diff
@@ -90,7 +94,6 @@ func New(conf Config) (*Peer, error) {
 
 	opts := []libp2p.Option{
 		libp2p.ConnectionManager(conf.ConnManager),
-		libp2p.DefaultTransports,
```

Comment:
Also important change. Now `libp2p.DefaultTransports` is automatically added by libp2p. Keeping this creates an error when initializing the libp2p host.